src/Plugin/Block/FooterBlock.php

```diff
@@ -104,10 +103,10 @@ public function build() {
       ],
     ];
 
-    // By some reasons drush config import does not invalidate cache tags.
-    // So for deployment we should take care about this case and
-    // perform cache invalidation
-    // via the "config:oe_corporate_blocks.data.footer" cache tag.
+    // Drush config import does not seem to invalidate cache tags.
+    // When syncing configuration we should take care of this case and
+    // invalidate the cache via the
+    // "config:oe_corporate_blocks.data.footer" cache tag.
```

Comment:
This comment is not needed. In any moment we import config we also clear the cache. Such as a deployment step.